website/main/data/authors/adlrocha.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Alfonso de la Rocha",
+  "image": "https://gravatar.com/adlrocha5feaaaf161?size=256",
+  "social": [
+    {
+      "linkedin": "https://linkedin.com/in/adlrocha"
+    },
+    {
+      "twitter": "https://twitter.com/adlrocha"
+    },
+    {
+      "github": "https://github.com/adlrocha"
+    },
+    {
+      "email": "mailto:me@adlrocha.com"
+    }
+  ]
+}
```

Please configure IDE to add new line to the end of the files automatically:
```suggestion
}

```